packages/synapse-react-client/src/components/RecentPublicationsGrid/RecentPublicationsGrid.tsx

```diff
@@ -96,7 +115,21 @@ function RecentPublicationsGrid(props: RecentPublicationsGridProps) {
                 padding: '20px 0px',
               }}
             >
-              {pub.values[titleColIndex]}
+              <MuiLink
+                href={convertDoiToLink(pub.values[doiColIndex] || '')}
+                target="_blank"
+                sx={{
+                  color: 'inherit',
+                  textDecoration: 'none',
+                  fontWeight: 'inherit',
+                  '&:hover': {
+                    color: '#1C9F87',
```

Comment:
This is not reusable across portals, and IMO this is indistinguishable from the default hover color on the EL portal.

If we need this color, please add it to the ELITE Portal Home page theme or ELITE Portal CSS overrides.
```suggestion
```